.github/workflows/bump-golang.yml

```diff
@@ -21,3 +21,6 @@ jobs:
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: ./.ci/bump-golang.yml
+          command: '--experimental apply'
+        env:
+          GITHUB_ACTOR: ${{ github.actor }}
```

Comment:
According to https://docs.github.com/en/actions/learn-github-actions/variables the `GITHUB_ACTOR` env is already set by default

Reply:
Good catch, I'm gonna try this in my playground and if it works then I'll remove it

Reply:
Given https://github.com/v1v/apm-server/commit/6b2f1cd6ee49309614c10c0b990644736a7fde18
When I run the workflow
Then it uses the implicit env variable GITHUB_ACTOR

See https://github.com/v1v/apm-server/pull/44